src/main/java/seedu/exchangecoursemapper/command/AddCoursesCommand.java

```diff
@@ -1,23 +1,37 @@
 package seedu.exchangecoursemapper.command;
 
+import seedu.exchangecoursemapper.constants.Logs;
 import seedu.exchangecoursemapper.storage.Storage;
 import seedu.exchangecoursemapper.exception.Exception;
+import seedu.exchangecoursemapper.constants.Assertions;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 
 public class AddCoursesCommand extends Command {
 
+    private static final Logger logger = Logger.getLogger(AddCoursesCommand.class.getName());
+
     Storage storage = new Storage();
 
     @Override
     public void execute(String userInput) {
         try {
+            logger.log(Level.INFO, Logs.TRIM_STRING);
             String description = trimString(userInput);
+            logger.log(Level.INFO, Logs.PARSE_ADD_COMMANDS);
             String[] descriptionSubstrings = parseAddCommand(description);
 
+            assert descriptionSubstrings.length == 3: Assertions.MISSING_FIELDS;
```

Comment:
I like the assertion here to make sure that the code is parsing the right input, to avoid accessing a null value!